lib/tests/streamlit/elements/bokeh_test.py

```diff
@@ -26,16 +27,29 @@
 class BokehTest(DeltaGeneratorTestCase):
     """Test ability to marshall bokeh_chart protos."""
 
+    @unittest.skipIf(
+        is_version_less_than(np.__version__, "2.0.0") is False,
+        "This test only runs if numpy is < 2.0.0",
```

Comment:
Could you please add an explanation why this fails, because from the test code itself its not obvious what role `numpy` is playing here (its not used directly)? And do you plan to add tests to close the gap of test coverage?

Reply:
I added additional information. tldr: the bokeh version supported by Streamlit uses numpy functionality removed in numpy 2.x. But since bokeh is an optional dependency, this isn't really a problem for supporting numpy 2.x.

Reply:
I added additional information. tldr: the bokeh version supported by Streamlit uses numpy functionality removed in numpy 2.x. But since bokeh is an optional dependency, this isn't really a problem for supporting numpy 2.x.

Reply:
Sounds good, thanks! The new comment is clear now 👍